sdk/src/Services/CloudFront/Custom/AmazonCloudFrontUrlSigner.cs

```diff
@@ -525,9 +525,6 @@ internal static RSAParameters ConvertPEMToRSAParameters(TextReader privateKeyRea
             {
                 throw new AmazonClientException("Invalid RSA Private Key", e);
             }
-
-            var rsa = RSA.Create();
```

Comment:
We call `ImportParameters` again here: https://github.com/aws/aws-sdk-net/blob/871aed7af1271bf39fa1992b4e29d51a1d13f809/sdk/src/Services/CloudFront/Custom/AmazonCloudFrontUrlSigner.cs#L509-L510

That method is invoked right after `ConvertPEMToRSAParameters`.

Reply:
You're right, it shouldn't but I decided not to change the interface of the signer in this PR. There's a separate issue (#3130) about improving performance of the CloudFront signers, that's probably a better place to discuss / design changes.

Another thing is that `ConcurrentDictionary` is not available in `net35`, which the SDK still supports: https://github.com/aws/aws-sdk-net/blob/main/sdk/src/Services/CloudFront/AWSSDK.CloudFront.Net35.csproj#L4